sbctl.go

```diff
@@ -214,14 +230,25 @@ func CreateBundle(bundle Bundle) {
 	defer os.Remove(tmpFile.Name())
 	bundle.Initramfs = tmpFile.Name()
 
-	GenerateBundle(&bundle)
+	return GenerateBundle(&bundle)
 }
 
-func GenerateAllBundles() {
+func GenerateAllBundles() error {
 	msg.Println("Generating EFI bundles....")
 	bundles := ReadBundleDatabase(BundleDBPath)
+	out := true
 	for _, bundle := range bundles {
-		CreateBundle(*bundle)
+		if !CreateBundle(*bundle) {
+			out = false
```

Comment:
You wouldn't know which bundle fails in this case. That might be a bit confusing?

`Failed to write bundle /boot....` would maybe be better?

Reply:
Good idea. Still, bundle generation still has  a lot of errors to propagate. CombineFiles, for example, still calls `log.Fatal`.

Reply:
I printed the error as well, but it might be better to make GenerateBundle return an `error`.